setup.py

```diff
@@ -29,6 +29,8 @@
     'matplotlib<=3.0.2',
     'seaborn',
     'six',
+    'pathlib',
```

Comment:
Which Python version is Pyam supporting? `pathlib` should be part of Py3.4+ or so i think ...

Reply:
indeed, thanks for pointing this out (CI is failing for other reasons)